documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -293,10 +294,161 @@ If some use case(s) for an API point to "Off-net" scenarios and where Consumptio
+    ExpO->>ExpO: - Validate User Identifier<br>- (Opt) Map to Operator subscription Identifier e.g.: phone_number<br>- Set UserId (sub)
+    ExpO->>ExpO: Check legal basis of the purpose<br>e.g.: contract, legitimate_interest, consent, etc
+    opt If User Consent is required for the legal basis of the purpose  
+        ExpO->>Consent: Check if Consent is granted
+    end
+    ExpO->>ExpO: Create 3-legged access token
+    ExpO->>-BE: HTTP 200 OK<br>{"access_token": "{OperatorAccessToken}"}
+end
+BE->>ExpO: Access Operator CAMARA API<br>Authorization: Bearer {OperatorAccessToken}    
+ExpO->>ExpO: Decrypt OperatorAccessToken,<br>grants Access,<br>progresses request to API Backend,<br>gets API response  
+ExpO->>BE: CAMARA API Response
+Note over BE,FE: Response
+```
+
+**Flow description**:
+
+The JWT Bearer Flow enables an API Consumer (typically the Application Backend or Aggregator) to obtain a Three-Legged Access Token from the API Provider (Operator) by presenting a signed JWT assertion on behalf of the User. The flow proceeds as follows:
+
+**Pre-requisite:** The API Consumer must share its public key with the API Provider during onboarding, so the API Provider can verify the signature of the JWT assertion.
+
+The API Consumer selects the User Identifier. The User is identified by the `sub` claim in the JWT, which must be a unique identifier for the User in the Operator's system. As per the CAMARA Security and Interoperability Profile, the `sub` claim MUST be either a phone number prefixed by "tel:" or a TS.43 token prefixed by "operatortoken:".
+
+Example JWT assertion, which MUST be signed by the API Consumer and MAY be encrypted:
+
+```json
+{
+  "iss": "client_id",
+  "sub": "operatortoken:ey...",
+  "aud": "https://az.api-provider.com/token.oauth2",
+  "exp": 1504807731,
+  "iat": 1504804131,
+  "jti": "53f42eb1-b751-44b5-bada-6990e08f35ac",
+  "scope": "dpv:FraudPreventionAndDetection number-verification:device-phone-number:read"
+}
+```
+
+The API Consumer sends a POST request to the API Exposure Platform's token endpoint (Step 1), using the `urn:ietf:params:oauth:grant-type:jwt-bearer` grant type and including the signed JWT assertion. The assertion contains the required claims: `iss` (client_id), `sub` (User Identifier), `aud` (token endpoint URL), `exp`, `iat`, `jti`, and `scope` (including the declared Purpose and CAMARA scopes).
+
+Example Token Request:
+
+```
+POST /token.oauth2 HTTP/1.1
+Host: as.example.com
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Ajwt-bearer
+&assertion=eyJhbGciOiJFUzI1NiIsImtpZCI6IjE2In0.
+eyJpc3Mi[...omitted for brevity...].
+J9l-ZhwP[...omitted for brevity...]
+```
+
+Then the API Exposure Platform validates the JWT assertion (Steps 2-4):
+  - Verifies the signature using the API Consumer's public key.
+  - Validates the User Identifier and, if needed, maps it to the Operator's internal subscription identifier.
```

Comment:
```suggestion
  - Validates the API Subject and, if needed, maps it e.g. to the Operator's internal subscription identifier.
```

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/294/commits/d72ca592a0d3ad9949b86c864f16f15db2daf9d6

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/294/commits/d72ca592a0d3ad9949b86c864f16f15db2daf9d6